R/databricks.R

```diff
@@ -153,14 +153,20 @@ is_hosted_session <- function() {
 # Returns a sensible driver name even if odbc.ini and odbcinst.ini do not
 # contain an entry for the Databricks ODBC driver.
 default_databricks_driver <- function() {
-  # For Linux and macOS we can default to the shared library paths used by the
+  # For Linux and macOS we can default to known shared library paths used by the
   # official installers. On Windows we use the official driver name instead.
-  default_path <- ""
+  default_paths <- c("")
   if (Sys.info()["sysname"] == "Linux") {
-    default_path <- "/opt/simba/spark/lib/64/libsparkodbc_sb64.so"
+    default_paths <- c(
+      "/opt/rstudio-drivers/spark/bin/lib/libsparkodbc_sb64.so",
+      "/opt/simba/spark/lib/64/libsparkodbc_sb64.so"
+    )
   } else if (Sys.info()["sysname"] == "Darwin") {
-    default_path <- "/Library/simba/spark/lib/libsparkodbc_sbu.dylib"
+    default_paths <- c(
+      "/Library/simba/spark/lib/libsparkodbc_sbu.dylib"
+    )
```

Comment:
Nit: style.

```suggestion
    default_paths <- "/Library/simba/spark/lib/libsparkodbc_sbu.dylib"
```

Reply:
My driver is in `/Library/simba/spark/lib/libsparkodbc_sb64-universal.dylib`. Should we try that too?

Reply:
Alternatively, we could take a pattern-matching approach for the filename, and search expected directories for a match: `/libsparkodbc_sb.*\.(dylib|so)$/`

Reply:
Yeah, that sounds like a good idea.